ddtrace/contrib/starlette/patch.py

```diff
@@ -57,6 +38,15 @@ def patch():
 
     _w("starlette.applications", "Starlette.__init__", traced_init)
 
+    handlers = [
+        "Mount",
+        "Route",
+    ]
+
+    # Wrap all of the handlers
+    for handler in handlers:
+        _w("starlette.routing", "{}.handle".format(handler), traced_handler)
```

Comment:
```suggestion
    # Wrap all of the routing handlers
    _w("starlette.routing", "Mount.handle", traced_handler)
    _w("starlette.routing", "Route.handle", traced_handler)
```

Since we only have 2, we probably don't need a loop, same for unpatch